clockwork_web/templates/base.html

```diff
@@ -224,33 +224,33 @@ <h1><a data-bs-toggle="collapse" data-bs-target=".formCollapse" aria-expanded="f
 													<!-- official documentation-->
 													{% if D_cluster['official_documentation'] %}
 														<a href="{{D_cluster['official_documentation']}}" target="_blank">
-															<i class="fa-solid fa-layer-group" data-bs-toggle="tooltip" data-bs-placement="right" title="Cluster documentation"></i>
+															<i class="fa-solid fa-layer-group" data-bs-toggle="tooltip" data-bs-placement="right" title="{{ gettext('Cluster documentation') }}"></i>
 														</a>
 													{% else %}
 														<a class="disabled">
-															<i class="fa-solid fa-layer-group" data-bs-toggle="tooltip" data-bs-placement="right" title="No documentation available"></i>
+															<i class="fa-solid fa-layer-group" data-bs-toggle="tooltip" data-bs-placement="right" title="{{ gettext('No documentation available') }}"></i>
 														</a>
 													{% endif %}
 													<!-- Grafana -->
 													<a href="https://dashboard.server.mila.quebec/" target="_blank">
-														<i class="fa-solid fa-file-lines" data-bs-toggle="tooltip" data-bs-placement="right" title="Grafana cluster link"></i>
+														<i class="fa-solid fa-file-lines" data-bs-toggle="tooltip" data-bs-placement="right" title="{{ gettext('Grafana cluster link') }}"></i>
 													</a>
 													{% if D_cluster['status']['jobs_are_old'] %}
 														<span class="cluster-info cluster-warning">
-															<i class="fa-solid fa-triangle-exclamation" data-bs-toggle="tooltip" data-bs-placement="right" title="Latest job edition is too old (more than 30 days ago)"></i>
+															<i class="fa-solid fa-triangle-exclamation" data-bs-toggle="tooltip" data-bs-placement="right" title="{{ gettext('Latest job edition is too old (more than 30 days ago)') }}"></i>
```

Comment:
'Latest job edition is too old (more than 30 days ago)' sounds weird.

I would recommend 'Possible stale jobs. Most recent update was more than 30 days ago.'